src/ui/locales/id.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Alihkan atribusi",
```

Comment:
I assume toggle is a button to show or dismiss an information about attribution, so I think it is better translated as show (Tampilkan) in indonesia
```suggestion
    "AttributionControl.ToggleAttribution": "Tampikan atribusi",
```